common/src/Renderer/FaceRenderer.cpp

```diff
@@ -185,14 +186,24 @@ namespace TrenchBroom {
                         continue;
                     }
 
+                    // Quake 3 shaders without blend functions completely disregard the texture's alpha channel
+                    const bool disableBlend = texture != nullptr && texture->blendFunc().enable == Assets::TextureBlendFunc::Enable::No;
```

Comment:
As I mentioned above this should actually be `texture->blendFunc().enable != Assets::TextureBlendFunc::Enable::Default` because we don't want to do the `discard` when drawing translucent textures.